templates/blog/featured-articles.html

```diff
@@ -0,0 +1,15 @@
+{% block featured_articles %}
+
+<div class="p-strip--suru-background" id="articles-list">
+  <div class="u-fixed-width">
+    <h4 style="color: #fff">Featured articles</h4>
```

Comment:
You should control the text color by adding `is-dark` to the strip

Reply:
It doesn't work for this specific strip